rust/main/agents/relayer/src/msg/metadata/multisig/base.rs

```diff
@@ -94,11 +95,16 @@ pub trait MultisigIsmMetadataBuilder: AsRef<MessageMetadataBuilder> + Send + Syn
 
 #[async_trait]
 impl<T: MultisigIsmMetadataBuilder> MetadataBuilder for T {
-    async fn build(&self, ism_address: H256, message: &HyperlaneMessage) -> Result<Metadata> {
+    async fn build(
+        &self,
+        message: &HyperlaneMessage,
+        params: MessageMetadataBuildParams,
+    ) -> eyre::Result<Metadata> {
```

Comment:
Unnecessary prefix `eyre::`

```suggestion
    ) -> Result<Metadata> {
```